### PR TITLE
feat: new ui

### PR DESCRIPTION
**Pull request by secondsun**
## Motivation
The new UI work is complete enough that we can begin using its container.

## What
This pr *removes the old ui* and updates *docker-compose* to use the new UI as well as updates the *README* to prodive new instructions.

## Why
The old UI was written in angular with old versions of pattern fly. Additionally we want to refactor UPS into several smaller quarkus containers. This would also mean providing a UI container.  Therefore, we rewrote the old UI as a container first.

### Docker-compose

1. Run `mvn clean install` to create a new build
2. Start the docker-compose suite found in `docker-compose`
3. Navigate to `http://localhost:8081`
4. Confirm UPS works

### Standalone

1. Run `mvn clean install` to create a new build
2. Follow the instructions for starting the containers in  `README.adoc`
3. Navigate to `http://localhost:8081`
4. Confirm UPS works

### CODE REVIEW

**Comment by danielpassos** [comment]
`mvn clean install` Give me:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.jboss.aerogear.unifiedpush.migrator.MigratorTest
[/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/bin/mysqld][--no-defaults][--basedir=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush][--socket=mysql.sock][--port=56316][--datadir=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/data][--pid-file=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/data/MysqldResource.pid]
[MysqldResource] launching mysqld (mysql)
Exception in thread "mysql" com.mysql.management.util.WrappedException: java.io.IOException: Cannot run program "/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/bin/mysqld": error=86, Bad CPU type in executable
	at com.mysql.management.util.Exceptions.toRuntime(Exceptions.java:55)
	at com.mysql.management.util.Exceptions$Block.exec(Exceptions.java:89)
	at com.mysql.management.util.RuntimeI$Default.exec(RuntimeI.java:180)
	at com.mysql.management.util.Shell$Default.run(Shell.java:140)
	at java.base/java.lang.Thread.run(Thread.java:832)```

Do I need to have MySQL up and running?

**Comment by ziccardi** [comment]
> `mvn clean install` Give me:
> 
> ```
> -------------------------------------------------------
>  T E S T S
> -------------------------------------------------------
> Running org.jboss.aerogear.unifiedpush.migrator.MigratorTest
> [/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/bin/mysqld][--no-defaults][--basedir=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush][--socket=mysql.sock][--port=56316][--datadir=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/data][--pid-file=/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/data/MysqldResource.pid]
> [MysqldResource] launching mysqld (mysql)
> Exception in thread "mysql" com.mysql.management.util.WrappedException: java.io.IOException: Cannot run program "/private/var/folders/yv/j4jhk6m17lq2l5z7rf3sp6_m0000gn/T/unifiedpush/bin/mysqld": error=86, Bad CPU type in executable
> 	at com.mysql.management.util.Exceptions.toRuntime(Exceptions.java:55)
> 	at com.mysql.management.util.Exceptions$Block.exec(Exceptions.java:89)
> 	at com.mysql.management.util.RuntimeI$Default.exec(RuntimeI.java:180)
> 	at com.mysql.management.util.Shell$Default.run(Shell.java:140)
> 	at java.base/java.lang.Thread.run(Thread.java:832)
> ```
> 
> Do I need to have MySQL up and running?

Are you running from a Mac?

**Comment by danielpassos** [comment]
> Are you running from a Mac?

Yes, I'm running on Mac. Fresh install

**Comment by ziccardi** [comment]
> > Are you running from a Mac?
> 
> Yes, I'm running on Mac. Fresh install

Then that's a known issue: https://stackoverflow.com/questions/59086476/mysqldresource-isnt-working-with-macos-catalina

File: docker-compose/.env [hunk 1]
```diff
@@ -14,3 +14,4 @@ ARTEMIS_SERVICE_HOST=artemis
 ARTEMIS_SERVICE_PORT=61616
 AMQ_MAX_RETRIES=5
 AMQ_BACKOFF_SECONDS=5
+UPS_DISABLED=webpush,web_push
```
Comment (danielpassos) [inline docker-compose/.env:17]: What does it do?

**Review by danielpassos — COMMENTED** [review]
I see a diff UI on the _ standalone_ version. Here is the step by step I did.

# Docker-compose

```shell
mvn clean install
cd docker-compose
docker-compose -f docker-compose.yaml up
open http://localhost:8081
```
2020-10-01 at 15 15 49@2x

# Standalone

```shell
mvn clean install -DskipTests
docker run -p 9999:8080 -it aerogear/unifiedpush-configurable-container:latest
open http://localhost:9999
```

2020-10-01 at 15 16 51@2x

**Review by danielpassos — APPROVED** [review]

File: docker-compose/.env (hunk 1 above)
Comment (secondsun) [reply docker-compose/.env:17]: It disables endpoints per feature

**Pull request merged**